### DISCUSSION TITLE
#131 Letter W - Stuck at 99.9%

### DISCUSSION THREAD

**Discussion by ms-saltation** [cssbattle/forum#323] [Q&A] [answered]
Stuck at 99.9%, tried changing the radius and color values without success.
Does anyone has a hint for me ?

`<p b><p a><style>*+*{background:#E38F66;margin:70 95;color:62306D}p{margin:0}[b]{width:30;height:30;background:#62306D;border-radius:50%50%0 0;box-shadow:90px 0,180px 0#FFFBDB}[a]{width:60;height:100;border-radius:0 0 100px 100px;border:30px solid;border-top:0;filter:drop-shadow(90px 0#FFFBDA);background:#0000`

**Comment by aarthvaen** [comment, marked as answer]
It oddly works by changing [a] class border-radius values to: ``border-radius:0 0 99px 100px;``

It's quite hard to find an explanation for such anomalies where you'd expect near-to-perfect values to do the job, but sometimes CSSBattle's renderer loves such strange approaches as well. 😄

**Reply by ms-saltation** [reply]
Thank you !

**Reply by imMatheus** [reply]
I was stuck on it too, thanks 🙏
